src/Store.ts

```diff
@@ -0,0 +1,582 @@
+import * as arrayUtil from 'dojo-core/array';
+import * as aspect from 'dojo-core/aspect';
+import Evented from 'dojo-core/Evented';
+import { default as has, add } from 'dojo-core/has';
```

Comment:
might want to import this as `hasAdd` instead of just add to give it some context, thats what we have done in other places. Not a big deal though